x-pack/plugins/cases/public/components/create/description.test.tsx

```diff
@@ -45,29 +46,70 @@ describe('Description', () => {
   });
 
   it('it renders', async () => {
-    const result = appMockRender.render(
+    appMockRender.render(
       <MockHookWrapperComponent>
         <Description {...defaultProps} />
       </MockHookWrapperComponent>
     );
 
-    expect(result.getByTestId('caseDescription')).toBeInTheDocument();
+    expect(screen.getByTestId('caseDescription')).toBeInTheDocument();
   });
 
   it('it changes the description', async () => {
-    const result = appMockRender.render(
+    appMockRender.render(
       <MockHookWrapperComponent>
         <Description {...defaultProps} />
       </MockHookWrapperComponent>
     );
 
+    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+
     userEvent.type(
-      result.getByRole('textbox'),
+      description,
       `${specialChars.selectAll}${specialChars.delete}My new description`
     );
 
     await waitFor(() => {
       expect(globalForm.getFormData()).toEqual({ description: 'My new description' });
     });
   });
+
+  it('shows an error when description is empty', async () => {
+    appMockRender.render(
+      <MockHookWrapperComponent>
+        <Description {...defaultProps} />
+      </MockHookWrapperComponent>
+    );
+
+    const description = screen.getByTestId('euiMarkdownEditorTextArea');
+
+    userEvent.clear(description);
+    userEvent.type(description, '  ');
+
+    await waitFor(() => {
+      expect(screen.getByText('A description is required.')).toBeInTheDocument();
+    });
+  });
+
+  it('shows an error when description is too long', async () => {
+    const longDescription = Array(MAX_DESCRIPTION_LENGTH / 2 + 1)
```

Comment:
I'm confused 😄  why does `MAX_DESCRIPTION_LENGTH / 2` trigger an error like `maximum length is > MAX_DESCRIPTION_LENGTH`?

Reply:
haha.. because it is converting an array to string, so` ['a','a'].toString() = a,a`, and commas will also be considered as characters 😄

Reply:
Thank you! i was feeling so dumb trying to figure it out 😆